### PR TITLE
Added some automation to "neyebenziyorsun"

### PR DESCRIPTION
**Pull request by Lyceion**
Probably it exists some syntax errors. I think it can be fixed verry easly...

### CODE REVIEW

**Comment by katurkmen** [comment]
I added some comments, also updated the project. Thanks.

File: main.py [hunk 1]
```diff
@@ -1,6 +1,18 @@
 # KaanBOT is a turkish discord bot. It is programmed to work in a single server.
 
 # Importing packages.
+import os
+import requests
+
+try:
+  import requests
```
Comment (kaanturkmen) [inline main.py:8]: I did not really understand why you imported requests module in two different location. There are at line 4 and line 8.

File: main.py [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+except ImportError:
+  print("Trying to Install required module: requests\n")
+  os.system('python -m pip install requests')
```
Comment (kaanturkmen) [inline main.py:11]: I think "pip install -r requirements.txt" command is better way to import packages. I updated the projects and added requirements.txt just now, you can check it out.

File: main.py [hunk 3]
```diff
@@ -142,7 +154,7 @@ async def unban(ctx, *, member):
             return
 
 
-# Commands which give information about certain topics and usable by all member is given below this line. #
```
Comment (kaanturkmen) [inline main.py:145]: That was not a typo. I am wrapping text with two octothorps if they are a header text.

File: main.py [hunk 4]
```diff
@@ -169,10 +181,12 @@ async def senkimsin(ctx):
...
 
 # Custom command
+data = 'https://raw.githubusercontent.com/katurkmen/KaanBOT/master/main.py'
 @client.command()
 async def neyebenziyorsun(ctx):
-
-    await ctx.send('Tahminen şöyle bir şeye:\nhttps://giphy.com/gifs/code-coding-seamless-xT9IgzoKnwFNmISR8I')
+    buffer = requests.get(data).text
+    buffer = buffer.split('#')
+    await ctx.send('Tahminen şöyle bir şeye: \n```python\n#' + buffer[random.randint(5, len(buffer) - 1)] + "\n```')
```
Comment (kaanturkmen) [inline main.py:189]: I pretty liked the idea of that.

**Review by kaanturkmen — COMMENTED** [review]
I updated the project and made comments on your pull request.

**Pull request closed without merging**